src/main/java/com/google/idea/perf/vfstracer/VfsStatTable.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.idea.perf.vfstracer
+
+import com.intellij.ui.table.JBTable
+import com.intellij.util.ui.JBUI
+import java.awt.Font
+import javax.swing.ListSelectionModel
+import javax.swing.SortOrder
+import javax.swing.SwingConstants
+import javax.swing.table.AbstractTableModel
+import javax.swing.table.DefaultTableCellRenderer
+import javax.swing.table.JTableHeader
+import javax.swing.table.TableRowSorter
+
+private const val COL_COUNT = 3
+
+// Table columns.
+private const val FILE_NAME = 0
+private const val STUB_INDEX_ACCESSES = 1
+private const val PSI_ELEMENT_WRAPS = 2
+
+class VfsStatTableModel: AbstractTableModel() {
+    private var data: List<VirtualFileStats>? = null
+
+    fun setData(newData: List<VirtualFileStats>?) {
+        data = newData
+        fireTableDataChanged()
+    }
+
+    override fun getColumnCount(): Int = COL_COUNT
+
+    override fun getColumnName(column: Int): String = when (column) {
+        FILE_NAME -> "file name"
+        STUB_INDEX_ACCESSES -> "stub index accesses"
+        PSI_ELEMENT_WRAPS -> "psi wraps"
```

Comment:
A PSI wrap meant creating an instance of `PsiElement` to wrap over an `ASTNode`. Yeah, "PSI parse" could also work because most `PsiElement` creations come from wrapping over `ASTNode`s that have just finished parsing.